lib/src/metta/runner/builtin_mods/das.rs

```diff
@@ -39,15 +52,73 @@ impl ModuleLoader for DasModLoader {
     fn load_tokens(&self, target: &MettaMod, metta: Metta) -> Result<(), String> {
         let mut tref = target.tokenizer().borrow_mut();
         let metta_runner = Arc::new(move |program| metta.run(SExprParser::new(program)));
-        let new_das_op = Atom::gnd(NewDasOp { metta_runner });
+
+        let params = Arc::new(Mutex::new(Properties::default()));
+
+        let new_das_op = Atom::gnd(NewDasOp {
+            maybe_metta_runner: Some(metta_runner.clone()),
+            params: params.clone(),
+        });
         tref.register_token(regex(r"new-das!"), move |_| new_das_op.clone());
+
+        let das_join_network_op = Atom::gnd(DasJoinNetworkOp {
+            params: params.clone(),
+        });
+        tref.register_token(regex(r"das-join-network!"), move |_| {
+            das_join_network_op.clone()
+        });
+
+        let das_services_op = Atom::gnd(DasServicesOp {
+            params: params.clone(),
+        });
+        tref.register_token(regex(r"das-services!"), move |_| das_services_op.clone());
+
+        let das_service_status_op = Atom::gnd(DasServiceStatusOp {});
+        tref.register_token(regex(r"das-service-status!"), move |_| {
+            das_service_status_op.clone()
+        });
+
+        let set_params_op = Atom::gnd(SetDasParamOp {
+            params: params.clone(),
+        });
+        tref.register_token(regex(r"das-set-param!"), move |_| set_params_op.clone());
+
+        let get_params_op = Atom::gnd(GetDasParamsOp {
+            params: params.clone(),
+        });
+        tref.register_token(regex(r"das-get-params!"), move |_| get_params_op.clone());
+
+        let das_create_context_op = Atom::gnd(CreateContextOp {
+            params: params.clone(),
+        });
+        tref.register_token(regex(r"das-create-context!"), move |_| {
+            das_create_context_op.clone()
+        });
+
+        let das_evolution_op = Atom::gnd(EvolutionDasOp {
+            maybe_metta_runner: Some(metta_runner),
+            params: params.clone(),
+        });
+        tref.register_token(regex(r"das-evolution!"), move |_| das_evolution_op.clone());
+
+        let das_link_creation_op = Atom::gnd(LinkCreationDasOp {
+            params: params.clone(),
+        });
+        tref.register_token(regex(r"das-link-creation!"), move |_| {
+            das_link_creation_op.clone()
+        });
+
+        let das_helpers_op = Atom::gnd(DasHelpersOp {});
+        tref.register_token(regex(r"das-helpers!"), move |_| das_helpers_op.clone());
+
         Ok(())
     }
 }
 
 #[derive(Clone)]
 pub struct NewDasOp {
-    metta_runner: Arc<dyn Fn(String) -> Result<Vec<Vec<Atom>>, String>>,
+    params: Arc<Mutex<Properties>>,
+    maybe_metta_runner: Option<Arc<dyn Fn(String) -> Result<Vec<Vec<Atom>>, String>>>,
```

Comment:
Probably it is not a question about this PR but more general question. I see that MeTTa runner is passed into the `NewDasOp` instance and into `EvolutionOp` instance. But what is a use-case for passing it? What kind of code DAS integration layer evaluates inside?